Foundation/NSPathUtilities.swift

```diff
@@ -678,10 +690,24 @@ internal func _NSCreateTemporaryFile(_ filePath: String) throws -> (Int32, Strin
 }
 
 internal func _NSCleanupTemporaryFile(_ auxFilePath: String, _ filePath: String) throws  {
+#if os(Windows)
+    try auxFilePath.withCString(encodedAs: UTF16.self) { fromPath in
+        try filePath.withCString(encodedAs: UTF16.self) { toPath in
+            let res = PathFileExistsW(toPath)
+                ? ReplaceFileW(toPath, fromPath, nil, 0, nil, nil)
+                : MoveFileW(fromPath, toPath)
```

Comment:
Ugh, this is racy.  The path can be created in between `PathFileExistsW` and `MoveFileW`.

Reply:
I've changed this to use CopyFileW and then just delete the temp file.